pkg/async/notifications/factory.go

```diff
@@ -66,6 +66,8 @@ func GetEmailer(config runtimeInterfaces.NotificationsConfig, scope promutils.Sc
 			scope,
 			sesClient,
 		)
+	case common.Sandbox:
+		return implementations.NewSendGridEmailer(config, scope)
```

Comment:
IIUC, we'll create SendGridEmailer at line 50, so we don't need these two lines here.

Reply:
Yes, but you need to turn your config to 
```
emailServerConfig:
      serviceName: sendgrid
      apiKeyEnvVar: "SENDGRID_API_KEY"
```
with one more key in serviceName